app/assets/javascripts/discourse/app/lib/intercept-click.js

```diff
@@ -18,29 +18,33 @@ export function wantsNewWindow(e) {
   This jQuery code intercepts clicks on those links and routes them properly.
 **/
 export default function interceptClick(e) {
-  if (wantsNewWindow(e)) {
+  const target = e.target.closest("a");
```

Comment:
What if the target **is** the `<a` element itself.

IIRC `.closest()` will never return itself. So maybe we need something like

```
const target = e.target.matches("a") ? e.target : e.target.closest("a");
```